### DISCUSSION TITLE
Possible adjustment on ltsp.conf man page for multiseat?

### DISCUSSION THREAD

**Discussion by jmgibson1981** [ltsp/ltsp#318] [General]
I implemented multi-seat on my main desktop. The wife needs a computer only occasionally so it made the most sense as we had some spare monitors and keyboards / mice. I had a difficult time getting it working. This is where I had my problem.

> To implement multiseat, where an LTSP client might have 2 or more seats, with separate monitors, keyboard and mice, the following section can be INCLUDEd. The "1" number maps the rule to "seat-1", while the rest of the parameter name ("GRAPHICS" etc) is ignored. You can check which hardware was assigned to which seat with loginctl seat-status seat0.
> 
> [multiseat]
> UDEV_SEAT_1_GRAPHICS="*/pci*/*/0000:01:00.0*"
> UDEV_SEAT_1_SOUND="*/sound/card1*"
> UDEV_SEAT_1_EVEN_USB_PORTS="*/usb?/?-[2,4,6,8,10,12,14,16,18]/*"

I was unsure of how to get the device names, ended up with loginctl. Easy with a sudo capable user in the chroot. Example below.

`/sys/devices/pci0000:00/0000:00:01.0/0000:01:00.1/sound/card1`

However this line didn't work. This one did.

`/devices/pci0000:00/0000:00:01.0/0000:01:00.1/sound/card1`

I've never messed with multi-seat before this, and it is a small difference. Maybe this is common knowledge? Or maybe I just had a total bad couple days and it should be obvious? I'm not certain. Either way maybe worth noting this somehow in the documentation?

**Comment by alkisg** [comment]
Hi, you forgot the wildcards (* and ?), see the ltsp.conf man page:

```shell
[multiseat]
UDEV_SEAT_1_GRAPHICS="*/pci*/*/0000:01:00.0*"
UDEV_SEAT_1_SOUND="*/sound/card1*"
UDEV_SEAT_1_EVEN_USB_PORTS="*/usb?/?-[2,4,6,8,10,12,14,16,18]/*"
```

**Comment by jmgibson1981** [comment]
Ok. Knew I missed something. I assumed that was just a placeholder for whatever came up. Thank you.
